.github/workflows/workflow.yaml

```diff
@@ -39,4 +39,36 @@ jobs:
           directory: ./coverage
           flags: unittests
           name: codecov-umbrella
-          fail_ci_if_error: false
\ No newline at end of file
+          fail_ci_if_error: false
+
+  docker:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: false
+          tags: valory/${{ github.event.repository.name }}:dev
+#      - name: Login to DockerHub
+#        uses: docker/login-action@v1
+#        if: github.ref == 'refs/heads/main'
+#        with:
+#          username: valory
+#          password: ${{ secrets.ACCESS_TOKEN }}
+#      - name: Push
+#        uses: docker/build-push-action@v2
+#        if: github.ref == 'refs/heads/main'
+#        with:
+#          context: .
+#          push: true
+#          tags: valory/${{ github.event.repository.name }}:main # note need to get the base ref to force for the branch
```

Comment:
we just need confirmation davidm that we want these image generated here.
We can also do it when you create a release